CONTRIBUTING.md

```diff
@@ -104,6 +104,15 @@ On Windows use the `Analyze Code Coverage for All Tests` on Visual Studio
 
 ![image](docs/vs-code-coverage.png)
 
+### Public Api
+
+Right now, we are using `Microsoft.CodeAnalysis.PublicApiAnalyzers` which will validate the public APi's. This analyzer will check if you changed a public property/method.
```

Comment:
```suggestion
It is critical to keep public API surface small and clean. This repository is using `Microsoft.CodeAnalysis.PublicApiAnalyzers` to validate the public APIs. This analyzer will check if you changed a public property/method so the change will be easily spotted in pull request. It will also ensure that OpenTelemetry doesn't expose APIs outside of the library primary concerns like a generic helper methods. 
```

Reply:
just updated!